dts/bindings/arm/nordic,nrf21540_gpio.yaml

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
```

Comment:
I guess it should be the Apache license, like in other such files. Also it would be good to shorten the header like it was done in other bindings.